keylime/api_version.py

```diff
@@ -8,11 +9,11 @@
 DEPRECATED_VERSIONS = ["1.0"]
 
 
-def current_version():
+def current_version() -> str:
     return CURRENT_VERSION
 
 
-def latest_minor_version(v):
+def latest_minor_version(v: str) -> str:
     v_obj = version.parse(str(v))
```

Comment:
This str(v) would probably enable floats and ints as well but so far callers only seem to pass in strings ...

Reply:
Once added the type maybe is safe to drop it call

Reply:
Added another patch now and also type-annotated also constants in this file now.

Reply:
The unittests failed due to floats and ints being used there. So, for now I adjusted the types to include floats and ints and didn't change the test cases.